validator/specs/lib/validateB3dmSpec.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var validateB3dm = require('../../lib/validateB3dm');
+
+describe('validateB3dm', function() {
+    it('validated is a b3dm tile', function() {
+        expect(validateB3dm(createB3dmTile())).toBe(true);
+    });
+
+    it('validated not b3dm tile, invalid magic', function() {
+        expect(validateB3dm(createInvalidMagic())).toBe(false);
+    });
+
+    it('validated not b3dm tile, invalid version', function() {
+        expect(validateB3dm(createInvalidVersion())).toBe(false);
+    });
+
+    it('validated not b3dm tile, wrong byteLength', function() {
```

Comment:
These test descriptions are a bit awkward. Instead try:

`returns true if the b3dm tile is valid`, `returns false if the b3dm has invalid magic`, etc.